Frends.MQTT.Receive/Frends.MQTT.Receive.Tests/UnitTests.cs

```diff
@@ -0,0 +1,154 @@
+namespace Frends.MQTT.Receive.Tests;
+
+using System.Threading.Tasks;
+using Frends.MQTT.Receive.Definitions;
+using NUnit.Framework;
+using MQTTnet;
+using System.Threading;
+using System;
+using System.Security.Authentication;
+using System.Net.Security;
+using MQTTnet.Protocol;
+
+/// <summary>
+/// Start broker with auto-generated TLS/auth:
+/// docker-compose up -d
+/// Run tests:
+/// dotnet test
+/// Clean up
+/// docker-compose down --volumes
+/// </summary>
+[TestFixture]
+internal class UnitTests
+{
+    /// <summary>
+    /// This test attempts to connect to an invalid broker address.
+    /// </summary>
+    /// <returns> Test succeeds when the connection is refused. </returns>
+    [Test]
+    public async Task Send_ShouldReturnErrorResult_WhenBrokerAddressIsInvalid()
+    {
+        var input = new Input
+        {
+            BrokerAddress = "invalid_address",
+            BrokerPort = 1883,
+            Topic = "test/topic",
+        };
+
+        var result = await MQTT.Receive(input, CancellationToken.None);
+
+        Assert.IsFalse(result.Success);
```

Comment:
Is there some error that we are waiting to get? E.g. it would be nice to check that the result tells us smth like "Invalid address" or other similar error text. Not just success false